experimenter/experimenter/features/tests/fixtures/valid_features/firefox-desktop/experimenter.yaml

```diff
@@ -32,24 +33,6 @@ missingVariables:
   exposureDescription: An exposure event
   isEarlyStartup: true
   variables: {}
-oldSetPrefFeature:
-  owner: owner@example.com
-  description: A feature that sets prefs via the old setPref schema
-  hasExposure: true
-  exposureDescription: An exposure event
-  variables:
-    string:
-      type: string
-      setPref: nimbus.test.string
-      description: String property
-    int:
-      type: int
-      setPref: nimbus.test.int
-      description: Integer property
-    boolean:
-      type: boolean
-      setPref: nimbus.test.boolean
-      description: Boolean property
```

Comment:
We still need to be able to load these features from old manifests *OR* we need to transform all the manifests in tree to use the new set-pref format.